src/config/global-config.c

```diff
@@ -56,6 +58,29 @@ int Read_GlobalSK(XML_NODE node, void *configp, __attribute__((unused)) void *ma
                 merror(XML_VALUEERR, node[i]->element, node[i]->content);
                 return (OS_INVALID);
             }
+            if (strcmp(node[i]->attributes[0], xml_ignore_frecuency) == 0) {
+                if (!OS_StrIsNum(node[i]->values[0])) {
+                    merror(XML_VALUEERR, node[i]->attributes[0], node[i]->values[0]);
+                    return (OS_INVALID);
+                }
+                Config->syscheck_ignore_frecuency = atoi(node[i]->values[0]);
+                if (Config->syscheck_ignore_frecuency < 1 || Config->syscheck_ignore_frecuency > 99) {
+                    merror(XML_VALUEERR, node[i]->attributes[1], node[i]->values[1]);
+                    return (OS_INVALID);
+                }
+            }
+            if (strcmp(node[i]->attributes[1], xml_ignore_time) == 0) {
+                if (!OS_StrIsNum(node[i]->values[1])) {
+                    merror(XML_VALUEERR, node[i]->attributes[1], node[i]->values[1]);
+                    return (OS_INVALID);
+                }
+                Config->syscheck_ignore_time = atoi(node[i]->values[1]);
+                if (Config->syscheck_ignore_time < 0 || Config->syscheck_ignore_time > 43200) {
```

Comment:
`syscheck_ignore_time` is unsigned, it won't be negative. Maybe you want to check that it is not zero (it means error).